base/process/internal_linux.cc

```diff
@@ -152,16 +159,19 @@ int64_t ReadStatFileAndGetFieldAsInt64(const FilePath& stat_file,
   return GetProcStatsFieldAsInt64(proc_stats, field_num);
 }
 
+#if !defined(STARBOARD)
 int64_t ReadProcStatsAndGetFieldAsInt64(pid_t pid, ProcStatsFields field_num) {
   FilePath stat_file = internal::GetProcPidDir(pid).Append(kStatFile);
   return ReadStatFileAndGetFieldAsInt64(stat_file, field_num);
```

Comment:
AFAIK this function should just work on our devices (and the related ones above). It probably was broken for a while until late last week because our GetThreadId returned incorrect values, but the change that broke that has been reverted.

You may only need pid_t to be defined (a sufficiently long, probably unsigned integer)